node/cmd/guardiand/adminclient.go

```diff
@@ -225,7 +226,7 @@ func runSignWormchainValidatorAddress(cmd *cobra.Command, args []string) error {
 	if !strings.HasPrefix(wormchainAddress, "wormhole") || strings.HasPrefix(wormchainAddress, "wormholeval") {
 		return fmt.Errorf("must provide a bech32 address that has 'wormhole' prefix")
 	}
-	gk, err := loadGuardianKey(guardianKeyPath)
+	gk, err := common.LoadGuardianKey(guardianKeyPath, *unsafeDevMode)
```

Comment:
will `unsafeDevMode` be set before here? asking because I'm not sure if the node flags apply here as well 🤔

Reply:
Turns out it is not, and the existing code did not handle devnet. I fixed it.